### DISCUSSION TITLE
Welcome to dartness Discussions!

### DISCUSSION THREAD

**Discussion by RicardoRB** [RicardoRB/dartness#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akamaotto** [comment]
I have been quietly following this project. I love NestJS and I love Dart. I hoped this would be my favourite Dart backend framework. But it seems like nothing is happening. It's sad if something this good should go away.

**Reply by RicardoRB** [reply]
Hey there! I really appreciate your comment, I stopped this project since I saw another backend projects were coming and also that some people weren't interested enough. 

Thanks to your comment, I will rethink to continue the project and keep pushing it, unfortunately just now I have personal problems to take care of, I can't promise anything, but I will check again the code and ask at least for some feedback ;)

**Reply by akamaotto** [reply]
Okay. I understand. I have seen the other backend projects and they are good. 
However, most of the backend projects don't come with strong OOP opinions embedded into the frameworks. 
I see them as problematic because most of them will work well for experienced developers who have already used frameworks like Spring but later felt that less-opinionated solutions like Express or Gin would work for them. 
To be honest, the discipline these persons get from Spring, Laravel, etc... helps them in their use of Express.

Many young developers get into Flutter as their first software engineering experience and they want to try Dart backend in order to take advantage of one language across frontend and backend. While that is good, these young developers do not understand OOP, Clean Code and Clean Architecture well. There is need for frameworks in Dart backend that enforces and informs the developer on principles of Object-Oriented Programming and aligns the team around a consistent architecture.

This will empower younger inexperienced teams to create powerful tools and reliable code.

Just my thoughts.
Cheers.

_And by the way, you definitely need to take care of yourself first. I am glad to assist in Community Development whenever (and if) you decide to get the project up again._
